frontend/pages/admin/UserManagementPage/UserManagementPage.tsx

```diff
@@ -373,21 +343,21 @@ const UserManagementPage = ({ router }: IUserManagementProps): JSX.Element => {
         .catch((userErrors: { data: IApiError }) => {
           if (userErrors.data.errors[0].reason.includes("already exists")) {
             setEditUserErrors({
-              email: "A user with this email address already exists",
+              email: userUpdatedEmailError,
             });
           } else if (
             userErrors.data.errors[0].reason.includes("required criteria")
           ) {
             setEditUserErrors({
-              password: "Password must meet the criteria below",
+              password: userUpdatedPasswordError,
             });
           } else {
-            renderFlash(
-              "error",
-              `Could not edit ${userEditing?.name}. Please try again.`
-            );
+            renderFlash("error", userUpdatedError);
           }
         })
+        .finally(() => {
+          setIsEditingUser(false);
```

Comment:
It's weird that the loading spinner is being controlled by `isEditingUser` rather than `isLoading`

Reply:
Also, I thought we were doing saving spinners on the buttons themselves and not on the pages?

Reply:
Yeah, I think that's the source of confusion.  Maybe `isSubmitting` or `isAwaiting` conveys it better?

Reply:
There really shouldn't be a _loading_ state for the entire modal since it's never waiting for a call to the backend to load, right? Should the fix match other UI where we added a saving state to the button? @lukeheath

Reply:
@gillespi314 should we push the fix because this is the intended behavior and open a new ticket to properly label the spinner and maybe update it so it's on the button?

Reply:
Ha, I was just typing up the same thing! Makes sense to me that we would use the button spinner here too. I'm good with opening a separate ticket for that since we have addressed the bug in the intended UX. And we can clarify the names of the variables then too.

Reply:
https://github.com/fleetdm/fleet/issues/6666